### DISCUSSION TITLE
PackagingCon 2023, 26-28 October, Berlin & Online

### DISCUSSION THREAD

**Discussion by tnabtaf** [Homebrew/discussions#4669] [Tap maintenance and Homebrew development]
Hi All,

This conference is relevant to the Homebrew community. If you wish to present your packaging or software supply chain work to the larger software packaging community, then please submit a proposal by ~~July 31~~ August 6.

Thanks, Dave C

---
## [PackagingCon 2023](https://packaging-con.org/)

* **26-28th October** 
* **Berlin and online**
* **Submission deadline:** ~~31st July~~ **6th August, anywhere on Earth**
* Early registration ends: 8th September

**[PackagingCon 2023](https://packaging-con.org/)** will bring together different software packaging ecosystems to share best practices and discuss common challenges in software packaging. From Python’s pip to Rust’s cargo to Julia’s Pkg, from Debian apt over Nix, to Homebrew, conda and mamba, and from vcpkg to Spack. The 2021 conference drew 350+ participants and [featured over 60 presentations](https://www.youtube.com/@packagingcon9302/videos).

### Call for Submissions

This conference is for developers of software package management tools, as well as software packagers and communities. Package management helps tame software complexity and establish and maintain software supply chain security. This is a challenging domain that embraces a diverse set of solutions. If you work in software package management then **please [share your work](https://cfp.packaging-con.org/2023/cfp) at PackagingCon! The call for submissions closes** ~~31st July~~ 6th August.** Topics from the most focused technical questions to community engagement are welcome.

### Register Now

**[Early registration](https://ti.to/packagingcon/packagingcon-2023) starts at €37.50 for virtual, and €150 for in person.** Significant discounts are available for early registration, students, and those paying out of their own funds. Early registration ends 8th September.

### Seeking Conference Sponsors

[PackagingCon 2023](https://packaging-con.org/) is a rare opportunity for your organization to support and reach the core decision makers and power users of software packaging across all languages and operating systems. [Multiple sponsorship levels are available](https://packaging-con.org/s/sponsorship_prospectus_packagingcon_2023.pdf) to suit your budget and goals. Please join sponsors such as [Quansight](https://quansight.com/) and [Anaconda](https://www.anaconda.com/) to help make PackagingCon 2023 possible and affordable.

We hope to see you in October, and your [presentation proposals](https://cfp.packaging-con.org/2023/cfp) this month!

**Comment by osalbahr** [comment]
I hope to see Homebrew presenting in 2023, as in 2021!

**Comment by tnabtaf** [comment]
An update:

The submission deadline has been extended to the end of Sunday, August 6, anywhere on Earth.

https://cfp.packaging-con.org/2023/cfp

**Comment by tnabtaf** [comment]
Hi All,

A [list of confirmed talks](https://cfp.packaging-con.org/2023/talk/) is now availabe.

And a reminder that [early registration closes September 8](https://packaging-con.org/)
